src/components/sidebar/sidebar-header/sidebar-header.spec.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import 'jest-styled-components';
+import SidebarHeader from './sidebar-header.component';
+import SidebarHeaderStyle from './sidebar-header.style';
+import Textbox from '../../textbox/textbox';
+import classicTheme from '../../../style/themes/classic';
+
+describe('Sidebar Header', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = shallow(
+      <SidebarHeader>
+        <Textbox />
+      </SidebarHeader>
+    );
+  });
+
+  describe('render', () => {
+    it('should render child', () => {
+      expect(wrapper.find(Textbox)).toBeTruthy();
+    });
+  });
+
+  describe('tags on component', () => {
+    it('include correct component, element and role data tags', () => {
+      wrapper.setProps({ 'data-element': 'bar', 'data-role': 'baz' });
```

Comment:
Is this testing anything at the moment? Looks like it's missing an assertion

Reply:
I removed that test, I think it's not necessary to test tags